src/main/java/uk/gov/gchq/hqdm/services/SpatioTemporalExtentServices.java

```diff
@@ -0,0 +1,1193 @@
+package uk.gov.gchq.hqdm.services;
+
+import uk.gov.gchq.hqdm.iri.HQDM;
+import uk.gov.gchq.hqdm.iri.IRI;
+import uk.gov.gchq.hqdm.iri.IriBase;
+import uk.gov.gchq.hqdm.model.*;
+import uk.gov.gchq.hqdm.model.impl.*;
+import uk.gov.gchq.hqdm.util.UID;
+
+/**
+ * Utilities for building and generating HQDM objects.
+ */
+public class SpatioTemporalExtentServices {
+    /**
+     * Generate a new PointInTime.
+     *
+     * @param eventTime Entity name of the point in time.
+     * @param pw The PossibleWorld the PointInTime is a part of.
+     * @param baseIri IriBase of the PointInTime.
+     * @return The generated PointInTime.
+     */
+    public static PointInTime event(final String eventTime, final PossibleWorld pw,
```

Comment:
```suggestion
    public static PointInTime event(final String eventTime, final PossibleWorld possibleWorld,
```

Reply:
No longer applies since the method is redundant and has been removed.